docker/Dockerfile-fargo-master

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.6
+
+ENV GOROOT=/usr/local/go
+
+RUN go get github.com/tools/godep
+RUN go get github.com/hudl/fargo
+
+WORKDIR /go/src/github.com/hudl/fargo/
+RUN godep restore
+
+#UNcomment if you want to run tests on master branch.
+#CMD go test -v ./...
```

Comment:
- [ ] If the point of this file is to run tests, why comment this out here?
- [ ] Ideally we'd have a way (perhaps via [`testing.Short`](https://golang.org/pkg/testing/#Short)) to avoid the network-dependent tests.